pkgs/by-name/ka/kapitano/package.nix

```
@@ -0,0 +1,124 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  meson,
+  ninja,
+  pkg-config,
+  glib,
+  gtk4,
+  libadwaita,
+  python3,
+  clamav,
+  appstream-glib,
+  desktop-file-utils,
+  libxml2,
+  gobject-introspection,
+  wrapGAppsHook4,
+  librsvg,
+}:
+stdenv.mkDerivation rec {
+  pname = "kapitano";
+  version = "1.1";
+
+  src = fetchurl {
+    url = "https://codeberg.org/zynequ/Kapitano/archive/${version}.tar.gz";
+    sha256 = "541ff3e04df2b8f602dfd7789d9ee90530b299c170f79bbaab4c4e12a8023a94";
+  };
```

Forgot this one: `fetchFromGitea` exists (example: https://github.com/NixOS/nixpkgs/blob/nixos-25.05/pkgs/by-name/ri/river/package.nix#L31-L38)
```suggestion
  src = fetchFromGitea {
    domain = "codeberg.org";
    owner = "zynequ";
    repo = "Kapitano";
    tag = finalAttrs.version;
    hash = "";
  };
```
(generate the hash from the error, it should start with `sha256-` and end with `=`)